extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -442,14 +442,20 @@ export class AmpConsent extends AMP.BaseElement {
     if (!isExperimentOn(this.win, 'amp-consent-geo-override')) {
       return this.getConsentRequiredPromiseLegacy_();
     }
-    if (typeof this.consentConfig_['consentRequired'] === 'boolean') {
-      return Promise.resolve(this.consentConfig_['consentRequired']);
-    } else {
-      return this.getConsentRemote_().then(consentInfo => {
-        const remoteResponse = consentInfo['consentRequired'];
-        return typeof remoteResponse === 'boolean' ? remoteResponse : true;
+    return this.consentStateManager_
+      .getLastConsentInstanceInfo()
+      .then(storedInfo => {
+        if (hasStoredValue(storedInfo)) {
+          return Promise.resolve(false);
```

Comment:
should it be `Promise.resolve(true)`?

Reply:
it doesn't matter what it resolves to, `initPromptUI` will get called, and it will do the local storage check again, and at that point it will decide to show or not show the UI. 

So maybe resolving to null is better.

Reply:
resolving to `true` is better for the current method name.

Reply:
Got it